### ISSUE TITLE
Support for background images

### ISSUE THREAD

**Issue by morten-leerhoy** [imazen/slimmage#14]
Would it be possible to add support for css added background images in stylesheets and inline?

**Comment by nathanaeljones** [comment]
Yes, we've refactored to make this possible as a plugin. It can't support the downlevel browsers we support, but it would be welcome as slimmage.background.js or the like within this repository, if we can setup tests in a clean manner.

**Comment by dancek** [comment]
I'm working on porting #37 as a plugin for the current unstable branch. Unfortunately I'm currently quite busy with other stuff so this might take time...

**Comment by fraabye** [comment]
@dancek did you find time for porting this to a plugin?

**Comment by dancek** [comment]
@fraabye, nope. Too low priority at work, too much other stuff at home. I might come back to this in autumn...

Obviously, feel free to port the code in the PR (but do keep the copyright notice intact).

**Comment by protherj** [comment]
FYI, I did submit a pull request for this here: https://github.com/imazen/slimmage/pull/57

**Comment by dancek** [comment]
@protherj great! On a quick glance the PR looks good, hope it won't be left hanging.
